pkg/controller.v1/pytorch/pytorchjob_controller.go

```diff
@@ -507,10 +507,10 @@ func (r *PyTorchJobReconciler) SetClusterSpec(job interface{}, podTemplate *core
 func (r *PyTorchJobReconciler) IsMasterRole(replicas map[kubeflowv1.ReplicaType]*kubeflowv1.ReplicaSpec,
 	rtype kubeflowv1.ReplicaType, index int) bool {
 	if _, ok := replicas[kubeflowv1.PyTorchJobReplicaTypeMaster]; ok {
-		return string(rtype) == strings.ToLower(string(kubeflowv1.PyTorchJobReplicaTypeMaster))
+		return rtype == kubeflowv1.PyTorchJobReplicaTypeMaster
```

Comment:
@andreyvelich  These checks will fail as we are converting type to lowercase in the following line
https://github.com/kubeflow/training-operator/blob/ab5de23608c4f14a8629cc84d2be55cf2b251f37/pkg/controller.v1/common/pod.go#L276